histogram/Cargo.toml

```diff
@@ -9,10 +9,13 @@ homepage = "https://github.com/pelikan-io/rustcommon/histogram"
 repository = "https://github.com/pelikan-io/rustcommon"
 
 [dependencies]
+serde = "1.0.144"
```

Comment:
Should we make serde an optional feature? This library is pretty light in terms of dependencies otherwise.

Reply:
I didn't know this was an option. Let's do that.